### PR TITLE
Added LICENSE for distribution/public use

### PR DESCRIPTION
**Pull request by jhdcruz**
**Non-software License** (https://choosealicense.com/non-software/)
> Full info: https://spdx.org/licenses/CC-BY-4.0.html

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,395 @@
+Attribution 4.0 International
```
Comment (jhdcruz) [inline LICENSE.txt]: Should be `Creative Commons Attribution 4.0 International`

Comment (jhdcruz) [inline LICENSE.txt]: Should be `Creative Commons Attribution 4.0 International`

Comment (jhdcruz) [reply LICENSE.txt]: Fixed in `commit b46018759d566b21363c19cc23530dca8cf682f0`

**Comment by SilverousBlack** [comment]
Should there be really a license, when I want it to be open source, since it's only library...???

**Pull request merged**